plugins/LilyPad/LilyPad.rc

```diff
@@ -58,82 +58,84 @@ BEGIN
+	CONTROL         "Axis 4", IDC_FF_AXIS4_ENABLED, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 205, 171, 91, 10
+	CONTROL         "Flip", IDC_FF_AXIS4_FLIP, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 302, 171, 35, 10
+	EDITTEXT        IDC_FF_AXIS4_SCALE, 375, 171, 33, 12, ES_RIGHT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP, WS_EX_RTLREADING
+	CONTROL         "", IDC_FF_AXIS5, "msctls_trackbar32", WS_TABSTOP, 199, 187, 214, 17
+	CONTROL         "Axis 5", IDC_FF_AXIS5_ENABLED, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 205, 207, 91, 10
+	CONTROL         "Flip", IDC_FF_AXIS5_FLIP, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 302, 207, 35, 10
+	EDITTEXT        IDC_FF_AXIS5_SCALE, 375, 207, 33, 12, ES_RIGHT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP, WS_EX_RTLREADING
+	CONTROL         "", IDC_FF_AXIS6, "msctls_trackbar32", WS_TABSTOP, 199, 223, 214, 17
+	CONTROL         "Axis 6", IDC_FF_AXIS6_ENABLED, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 205, 243, 91, 10
+	CONTROL         "Flip", IDC_FF_AXIS6_FLIP, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 302, 243, 35, 10
+	EDITTEXT        IDC_FF_AXIS6_SCALE, 375, 243, 33, 12, ES_RIGHT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP, WS_EX_RTLREADING
+	PUSHBUTTON      "Test", ID_TEST, 196, 263, 59, 15
+	PUSHBUTTON      "Back to Controls", ID_CONTROLS, 196, 299, 59, 15
+	GROUPBOX        "Configure Binding", ID_SENSITIVITY, 195, 189, 222, 70
+	EDITTEXT        IDC_AXIS_DEVICE1, 202, 202, 74, 12, ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP //+7
+	COMBOBOX        IDC_AXIS_DIRECTION, 276, 200, 70, 47, CBS_DROPDOWNLIST | WS_TABSTOP
+	EDITTEXT        IDC_AXIS_CONTROL1, 349, 202, 65, 12, ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP
+	LTEXT           "Sensitivity", IDC_LABEL_SENSITIVITY, 202, 218, 42, 8
+	CONTROL         "Turbo", IDC_TURBO, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 202, 228, 34, 10
+	CONTROL         "", IDC_SLIDER1, "msctls_trackbar32", WS_TABSTOP, 240, 218, 131, 17
+	EDITTEXT        IDC_AXIS_SENSITIVITY1, 377, 216, 33, 12, ES_RIGHT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP, WS_EX_RTLREADING
+	CONTROL         "Flip", IDC_FLIP1, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 385, 227, 27, 10
+	LTEXT           "Dead Zone", IDC_LABEL_DEADZONE, 202, 243, 42, 8
+	CONTROL         "", IDC_SLIDER_DEADZONE, "msctls_trackbar32", WS_TABSTOP, 240, 239, 131, 17
+	EDITTEXT        IDC_AXIS_DEADZONE, 377, 242, 33, 12, ES_RIGHT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP, WS_EX_RTLREADING
+	PUSHBUTTON      "Lock Input", ID_LOCK, 196, 263, 59, 15
+	PUSHBUTTON      "Lock Direction", ID_LOCK_DIRECTION, 196, 281, 59, 15
+	PUSHBUTTON      "Lock Buttons", ID_LOCK_BUTTONS, 196, 299, 59, 15
+	GROUPBOX        "Add Force Feedback Effect", IDC_STATIC, 262, 263, 155, 51
+	COMBOBOX        IDC_FORCEFEEDBACK, 269, 276, 142, 106, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
+	PUSHBUTTON      "Big Motor", ID_BIG_MOTOR, 269, 294, 64, 14
+	PUSHBUTTON      "Small Motor", ID_SMALL_MOTOR, 347, 294, 64, 14
```

Comment:
Hand edited ? 

According to **Rule #7834** on turtleli's book , spaces between other arguements should be removed on resource script files.

**Example**:
``   "Small Motor", ID_SMALL_MOTOR, 347, 294, 64, 14``  **->**   ``Small Motor",ID_SMALL_MOTOR,347,294,64,14``